### ISSUE TITLE
PreTestTests very slow

### ISSUE THREAD

**Issue by Jameskmonger** [alsatian-test/alsatian#238]
In the .gif in the readme, the test `multipleTestFixtureWithMultipleTestsWithMultipleTestCases...` runs very very slowly. It looks like it's more of an integration stress test, which I'm not sure belongs in the unit tests.

**Comment by jamesrichford** [comment]
Yeah, probably want to put them in a stress tests folder under integration tests I'd suspect but not until post 1.0.0 - any other stress tests you can think of to do @Jameskmonger?

**Comment by jamesrichford** [comment]
I think the reason out of interest they are in there is there was a rework of the test directory when we added integration-tests and these must have been mistakenly put in that folder

**Comment by jamesrichford** [comment]
Also we should totally replace the video of the test run when those are out! Even more awesome 😄
